### DISCUSSION TITLE
EJS Problem

### DISCUSSION THREAD

**Discussion by Day-Codes** [vercel/community#6804] [Help]
### Summary

everytime i run EJS on vercel i get the error 404: NOT_FOUND
Code: NOT_FOUND
ID: cle1::ff6sd-1715382735956-308ee1bfceba"

### Steps to Reproduce

https://portfolioconvert.vercel.app (this is a vercel template)

**Comment by versecafe** [comment]
@Day-Codes Can you share a link to the template repository?

**Reply by Day-Codes** [reply]
https://github.com/Day-Codes/portfolioconvert

**Reply by versecafe** [reply]
@Day-Codes Either the link is wrong or the repository is private, can you double check?

**Reply by sahilatahar** [reply]
@Day-Codes You repo is private make it public
